frontend/src/components/AllowedItems.js

```diff
@@ -1,39 +1,89 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { ThemeProvider } from '@material-ui/core/styles';
 import Button from "@material-ui/core/Button";
 import TextField from '@material-ui/core/TextField';
 import axios from 'axios';
 import theme from '../theme';
-import { makeStyles } from '@material-ui/core/styles';
-
-export default function AllowedItems() {
-  const classes = useStyles();
-
-  return (
-    <ThemeProvider theme={theme}>
-      <TextField
-        id="outlined-multiline-static"
-        multiline
-        rows="10"
-        className={classes.textField}
-        placeholder="List all allowed items or any other notes about your composting bin"
-        margin="normal"
-        variant="outlined"
-      />
-      <br />
-      <br />
-      <Button
-        variant="contained"
-        color="primary"
-      >Save</Button>
-    </ThemeProvider>
-  )
+
+
+class AllowedItems extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allowedItems: ''
+    }
+  }
+
+  // componentDidMount() {
```

Comment:
This is WIP to allow these fields to be populated with the users current preferences when the page loads.